src/redefined.resolver.ts

```diff
@@ -21,19 +21,31 @@ export class RedefinedResolver {
 
     private resolvers: ResolverService[];
 
+    private config: Promise<ResolversParams> | undefined = undefined;
+
     constructor(
       private options?: ResolverOptions
     ) {
         if (options && !options.resolvers.length) {
             throw Error("“resolvers” option must be a non-empty array or falsy")
         }
 
+        if (!options && !this.config) {
```

Comment:
I think !this.config no needed in constructor cause constructor will be called once. (if this be in resolve !this.config will be must)